src/pmpo_createTestMPMesh.cpp

```diff
@@ -15,8 +15,8 @@ Mesh* createMesh(const mesh_type meshType, const geom_type geomType,
                  const std::vector<std::vector<int>> &elm2ElmConn_array){
     const int nCells = nCells_size;
     const int nVertices = nVertices_size;
-    DoubleVec3dView vtxCoords("verticesCoordinates", nVertices);
-    DoubleVec3dView::HostMirror h_vtxCoords = Kokkos::create_mirror_view(vtxCoords);
+    MeshFView<MeshF_VtxCoords> vtxCoords("verticesCoordinates", nVertices);
```

Comment:
I don't understand this line: type. Need to check/discuss.

Reply:
At some point we talked about connecting the types between the mesh and material points class. So we attached the field type to the mesh field instead of hardcoding it.

Reply:
OK, I will review it again in next round.